### PR TITLE
Fix newsfeed holes left behind by hidden events

### LINKED ISSUE #475: Too much whitespace under some news feed events
screen shot 2017-06-06 at 17 32 04

### PR DESCRIPTION
**Pull request by abnersajr**
Fixes #475.

Some simple alerts have been hidden on CSS, however the empty space still there.
I inspect many whitespaces and for me, the only case what I could see was when someone creates a branch.

I also have added 'package-lock.json'. I saw that We do not have this yet. However, if necessary I can remove, just ping me and I push back without them.

### CODE REVIEW

**Comment by sindresorhus** [comment]
Any way to do this with only CSS?

---

> I also have added 'package-lock.json'. I saw that We do not have this yet. However, if necessary I can remove, just ping me and I push back without them.

Yeah, remove it. I don't want a lockfile here.

**Comment by abnersajr** [comment]
In this moment no. We don't have a parent selector that fill our necessity.
Microsoft are thinking on that. https://developer.microsoft.com/en-us/microsoft-edge/platform/status/cssrelationalpseudoclasshas/

**Comment by abnersajr** [comment]
Add npm and yarn lock files to gitignore

File: .gitignore [hunk 1]
```diff
@@ -2,3 +2,5 @@ node_modules
 extension/**/*.js
 extension/**/*.map
 !extension/jquery-3.slim.min.js
+package-lock.json
+yarn.lock
```
Comment (sindresorhus) [inline .gitignore]: This is unrelated to this PR and is already done in master.

**Comment by sindresorhus** [comment]
This PR needs a more descriptive title.

**Comment by sindresorhus** [comment]
This was indirectly fixed by https://github.com/sindresorhus/refined-github/pull/599.
